docs/jina-ai-cloud/login.md

```diff
@@ -76,19 +76,10 @@ Installed along with Jina, you can leverage the `hubble` package to manage login
 ```python
 import hubble
 
-# Open browser automatically and login via 3rd party.
-# Token will be saved locally.
-hubble.login()
-```
-
-#### Login from Jupyter notebook/Google Colab
-
-```python
-import hubble
-
-# Use Personal Access Token or browser to login.
-# Token will be saved locally.
-hubble.notebook_login()
+# Login via browser or Personal Access Token. The token will be saved locally.
```

Comment:
Add link to token section on cloud.jina.ai around "Personal Access Token"

Reply:
I've just realized we can't have a link inside the code block, what do you suggest? Add a link outside the code block or do not add a link?